flake.nix

```
@@ -80,7 +80,9 @@
             packages =
               prev.packages
               ++ (
-                with common.pkgs; [lld_13 lldb cargo-tarpaulin cargo-flamegraph rust-analyzer]
+                with common.pkgs;
+                [lld_13 lldb cargo-flamegraph rust-analyzer] ++
+                (lib.optional stdenv.isx86_64 cargo-tarpaulin)
```

```suggestion
                (lib.optional (stdenv.isx86_64 && stdenv.isLinux) cargo-tarpaulin)
```

Based on

> Tarpaulin only supports x86_64 processors running Linux

from cargo-tarpaulin's [readme](https://github.com/xd009642/tarpaulin/blob/79c113698f2960b47496c4953ef33f7d8c8b96c1/README.md?plain=1#L17)